fele-client-service/utils/db.js

```diff
@@ -27,7 +27,26 @@ const deleteDatabase = async(databaseName) => {
 }
 
 const insertToDatabase = async(databaseName, documentToBeInserted) => {
-    couch.insert(databaseName, documentToBeInserted);
+    try{
+        const { data } = await couch.insert(databaseName, documentToBeInserted)
+        console.info("Channel created successfully!")
+        return data.id
+
+    } catch(error) {
+        console.err("Error inserting data: ", error)
+            return false
+    }
+}
+
+const checkIfNetworkExists = async (databaseName) => {
+    console.info(`checking if ${databaseName} DB exists....`)
+    const dbs = await couch.listDatabases()
```

Comment:
is there any find method in couchDB ?

Reply:
I have found `listDatabases()` in the documentation to retrieve database information. If there is any `find()` method to retrieve the database names I will use it. Not sure if there is any. If you find any please let me know